### DISCUSSION TITLE
Is there a way to make a new element

### DISCUSSION THREAD

**Discussion by Tarang** [gonzalezreal/swift-markdown-ui#270] [Q&A]
Is there a way to define my own element and what it would look like to parse:, e.g if i went [Button], to be able to use a new .Button block?

It can be quite handy to extend the markdown syntax

**Comment by Tarang** [comment]
Another one might be [1] (as a citation reference, to use a smaller font). Was wondering how i could add a custom block

**Comment by gonzalezreal** [comment]
Hi @Tarang,

MarkdownUI supports [GitHub Flavored  Markdown Spec](https://github.github.com/gfm/), and we don't have plans to add any extensibility.

**Comment by axiel7** [comment]
If I open a PR with this feature, would you consider merging it?

**Reply by reubenjs** [reply]
how would you go about doing something like this? I'm interested in adding custom elements too. just starting with swift though
